### DISCUSSION TITLE
In book format PDF generates latex output in the first chapter folder with broken image references

### DISCUSSION THREAD

**Discussion by phish108** [quarto-dev/quarto-cli#10810] [Q&A]
### Description

Hi all

I am a bit confused with the latest PDF handling and need some help: A year ago (with quarto 1.3) I was able to render [my quarto document code](https://github.com/dxiai/daten-organisation)  as HTML and PDF just fine. Now I wanted to recompile the very same document with quarto 1.5.57 (or 1.6.9) and quarto fails during PDF generation with  the following latex error:

```
Unable to load picture or PDF file 'kapitel/01-einleitung/figures/by-nc-sa.png'.
<to be read again>
                   }
l.315 ...itel/01-einleitung/figures/by-nc-sa.png}}
```

If I exclude PDF output, the document renders just fine. 

The error means that the first image is not present at the referenced location. If I remove that image, the error skips to the next image.

What I figured is that the latex output is created in the first chapter folder rather than in the `output-dir` or in the `_freeze` folder. It appears that latex is also run from within this folder, which explains why the references to the any image must break as they are referred in the generated document as if latex runs from the root folder, which seems not to be the case.

I stripped my `_quarto.yaml` file to minimal cases but the problem persists. I also deleted all cached files in `_book`, `.quarto`, and `_freeze` without success.

As I could not find bug reports on this behavior, I guess that am missing  something non-obvious but important related to PDF generation from the docs. There are old discussions and issues, but they were using bad file references, which is not the case for my document.

My tools are freshly updated: 

```
❯ quarto tools
[✓] Inspecting tools

Tool         Status         Installed     Latest
chromium     Up to date     869685        869685
tinytex      Up to date     v2024.09      v2024.09
```

My document source is [fully available](https://github.com/dxiai/daten-organisation).

Any help is appreciated.

Christian.

**Comment by mcanouil** [comment]
Unfortunately, we can't really help you with something that big, we don't have the time to debug your project.
You need to do that yourself, i.e., strip pieces one by one until you find the document with the issue, then the line. Possibly, use `keep-tex: true` to look at the raw LaTeX code.

**Reply by phish108** [reply]
@mcanouil thank you :(

**Reply by phish108** [reply]
I ran more checks on my other projects. 

Just to let you know.

The behaviour is consistent with book projects with chapters in separate folders, both, with the 1.5 release and the 1.6 release.

- The image paths are generated in the tex file as expected anchored to the root of the project. 
- The main tex file is generated in the first chapter's folder, which is not expected, but is not a problem by itself. 
- The first chapter's folder is used as root during the tinytex build process, which breaks all paths of the included files because they are anchored to the project's root. 

This behaviour looks wrong to me. 

I will provide a minimal test case and file an issue soon.
